functions/src/venue.ts

```diff
@@ -427,9 +386,15 @@ export const setAuditoriumSections = functions.https.onCall(
 
     const { venueId, numberOfSections } = data;
 
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
-    await checkUserIsOwner(venueId, context.auth.token.user_id);
+    const space = await getSpaceById(venueId);
+
+    await throwErrorIfNeitherWorldNorSpaceOwner({
+      spaceId: venueId,
+      worldId: space.worldId,
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      uid: context.auth.token.user_id,
```

Comment:
I don't think it would work as context.auth would still be marked as `object | undefined`

Reply:
Hm... didn't checkout the code to see the actual message, I assumed it might be the one having underscore in it. The way `o deal with possible undefined value is  simpler I think, anything from `as unknown as object` assertion to `??` to type guards... 

If it is OOS, might benefit from adding `@debt clean up ignore comments` as well, where there are 2 there can be 3 lines of comment